### DISCUSSION TITLE
Kommunikation zwischen RaspberryMatic und HmIP-RFUSB gestört

### DISCUSSION THREAD

**Discussion by espamik** [OpenCCU/OpenCCU#1953] [🇩🇪 Fragen&Antworten] [answered]
### Describe the issue you are experiencing

Bin absoluter Neuling bei HM und habe entsprechend der Anleitung auf einem Raspi 3 RaspberryMatic unter Docker mit einem HmIP-RFUSB installiert. Betriebssystem ist Raspbian GNU/Linux 11 mit Kernel 5.15.61-v7+

Im Webinterface wird der HmIP-RFUSB Stick angezeigt, ich kann jedoch keine Geräte anlernen.

### Describe the behavior you expected

Auf der Startseite wird bei Duty Cycle 0% und bei Carrier Sense 100% angezeigt. In etlichen Threads habe ich über Funkstörungen gelesen und etliche Ortsänderungen vorgenommen, ohne dass sich etwas geändert hat.

Es erscheint regelmäßig eine Alarmmeldung, die bestätigt werden muss:
Alarmname: WatchDog: carriersense-CCU
Status: ausgelöst
Beschreibung: CarrierSense 100% (1F2995E834)

### Steps to reproduce the issue

1. Installation unter Docker nach der Anleitung [https://github.com/jens-maus/RaspberryMatic/wiki/Installation-Docker-OCI]
2. Wenn ich im Webinterface „Test starten“ klicke, färbt sich die Zeitanzeige gelb. Es erfolgt keine weitere Aktion, ich kann das Fenster jedoch wieder schließen. Ich vermute, dass die Kommunikation zwischen RaspberryMatic und dem HmIP-RFUSB gestört ist.

### What is the version this bug report is based on?

3.65.8.20220831 

### Which base platform are you running?

rpi3 (RaspberryPi3)

### Which HomeMatic/homematicIP radio module are you using?

HmIP-RFUSB

### Anything in the logs that might be useful for us?

```txt
Starting watchdog...
Identifying host system: oci, OK
Initializing RTC Clock: no hardware found
Running sysctl: OK
Checking for Factory Reset: not required
Checking for Backup Restore: not required
Initializing System: OK
Setup ca-certificates: OK
Starting logging: OK
Populating /dev using udev: done
Init onboard LEDs: init, OK
Starting irqbalance: OK
Starting iptables: OK
Starting network: eth0: link up, fixed, firewall, inet up, 172.21.0.6, OK
Identifying Homematic RF-Hardware: ....HmRF: n/a, HmIP: HMIP-RFUSB/USB, OK
Updating Homematic RF-Hardware: HMIP-RFUSB: 4.4.18, not necessary, OK
Starting hs485dLoader: disabled
Starting xinetd: OK
Starting eq3configd: OK
Starting lighttpd: OK
Starting ser2net: disabled
Starting ssdpd: OK
Starting NUT services: disabled
Initializing Third-Party Addons: OK
Starting LGWFirmwareUpdate: ...OK
Setting LAN Gateway keys: OK
Starting hs485d: disabled
Starting multimacd: not required
Starting rfd: no BidCos-RF hardware found
Starting HMIPServer: ................................OK
Starting ReGaHss: .OK
Starting CloudMatic: OK
Starting NeoServer: disabled
Starting Third-Party Addons: OK
find: /usr/local/etc/monit*.cfg: No such file or directory
Starting crond: OK
Setup onboard LEDs: booted, OK
Finished Boot: 3.65.8.20220831 (raspmatic_oci_arm)
```

### Additional information

Im Voraus vielen Dank für alle Tipps!

**Comment by jens-maus** [comment]
> Auf der Startseite wird bei Duty Cycle 0% und bei Carrier Sense 100% angezeigt.

Das ist das Problem. Der Carrier Sense *muss* dauerhaft < 10% (und nicht 100%) sein damit man überhaupt irgendwas angelernt bekommt. Ergo, den HmIP-RFUSB mit einem 2-3m Verlängerungskabel weit weg von jeglichen Funkstörquellen bringen.

**Comment by espamik** [comment]
Hallo, vielen Dank für die schnelle Hilfe. Das habe ich schon probiert. Der Stick ist an einem USB-Verlängerungskabel und ich habe im Umfeld alles ausgeschaltet, also den X64 Server, RPI4 etc. Ergebnis stets unverändert: Carrier Sense 100%
In den Logs kann ich nichts finden, was mir weiterhilft.

**Reply by jens-maus** [reply]
Da gibts auch nix im Log zu finden! CarrierSense=100% bedeutet das 868MHz Funkband ist komplett dicht in der näheren Umgebung der Funkantenne. Das deutet einfach auf einen starken Funkstörer hin! Dazu wird auch nix in irgendeinem Logfile gemeldet.

Einzige Lösung: Funkstörer finden und eliminieren!

**Comment by MichaelN0815** [comment, marked as answer]
Siehe https://homematic-forum.de/forum/viewtopic.php?f=31&t=75532

**Comment by espamik** [comment]
OK, dann mache ich mich nochmal auf die Suche nach dem Störer. Habt ihr noch einen Tipp bzgl. dem USB-Verlängerungskabel? Wie lang kann das maximal sein, damit der Stick noch zuverlässig funktioniert? Gibt es beim Kauf etwas bzgl. der Spezifikation zu beachten?

**Comment by espamik** [comment]
Sorry, ich muss noch mal nachhaken bzgl. meiner zweiten Frage:
Wenn ich im Webinterface „Test starten“ klicke, färbt sich die Zeitanzeige gelb. Es erfolgt keine weitere Aktion, ich kann das Fenster jedoch wieder schließen. Ich vermute, dass die Kommunikation zwischen RaspberryMatic und dem HmIP-RFUSB gestört ist.
Warum wird kein Test durchgeführt, lediglich die Zeitanzeige bleibt gelb und nichts passiert?

**Comment by MichaelN0815** [comment]
Dem link möchtest du sicherheitshalber nicht folgen? Angst vor Erkenntnis Gewinn?

**Comment by espamik** [comment]
Das Problem bzgl. des Carrier Levels konnte ich lösen. Nachfolgend hier meine Erfahrung zur Lösung des Problems in der Hoffnung, dass es anderen Nutzern hilft:

Etwaige Funkstörquellen konnte ich über etliche Tage hinweg nicht finden. Die Ursache für die Störung lag in meinem Fall wohl in der Kommunikation über den USB-Port, wobei ich den Fehler weder reproduzieren kann noch in den Logs etwas dazu gefunden habe:

Den Stick habe ich im oberen 2. USB Port von rechts in Betrieb genommen. Im rechten oberen USB Port ist per USB Adapter eine SSD eingesteckt, von dem der RPI3 bootet. Das Anmelden des Sticks in RaspberryMatic hat in diesem 2. Port von rechts funktioniert. 
Inspiriert durch eine Nachricht in einem anderen Forum habe ich den Stick und die SSD im jeweils anderen Port eingesteckt und siehe da, es funktioniert dauerhaft, auch nach Reboot und an verschiedenen Stellen im Haus. Auch nach einem Rücktausch funktioniert der Stick perfekt. Carrier Sense beträgt seitdem 0%.

@[MichaelN0815](https://github.com/MichaelN0815) Dem von dir geposteten Link bin ich gefolgt, der Thread war mir schon vorher bekannt und was dort empfohlen wurde, hatte ich schon geprüft, bevor ich diese Anfrage überhaupt gestellt hatte. Daher hatte ich auch das Log gepostet, da ich Funkprobleme ausgeschlossen und den Fehler auf Hardwareebene vermutet hatte. 
Auf dem Niveau deiner oberlehrerhaften Rückmeldung mit dem Verweis auf den Link möchte ich mich hier nicht austauschen.
